config/base.yml

```diff
@@ -474,63 +476,6 @@ Naming/UncommunicativeBlockParamName:
 Naming/VariableName:
   Enabled: true
 
-Performance/Caller:
-  Enabled: true
-
-Performance/CompareWithBlock:
-  Enabled: true
-
-Performance/Count:
-  Enabled: true
-
-Performance/Detect:
-  Enabled: true
-
-Performance/DoubleStartEndWith:
-  Enabled: true
-
-Performance/EndWith:
-  Enabled: true
-
-Performance/FixedSize:
-  Enabled: true
-
-Performance/FlatMap:
-  Enabled: true
-
-Performance/InefficientHashSearch:
-  Enabled: true
-
-Performance/RangeInclude:
-  Enabled: true
-
-Performance/RedundantMatch:
-  Enabled: true
-
-Performance/RedundantMerge:
-  Enabled: true
-
-Performance/RegexpMatch:
-  Enabled: true
-
-Performance/ReverseEach:
-  Enabled: true
-
-Performance/Size:
-  Enabled: true
-
-Performance/StartWith:
-  Enabled: true
-
-Performance/StringReplacement:
-  Enabled: true
-
-Performance/UnfreezeString:
-  Enabled: true
-
-Performance/UriDefaultParser:
-  Enabled: true
-
```

Comment:
I expected these to still be turned on -- can we leave them on?

Subtext: I suspect this might have been deleted if rubocop-performance is enabled-by-default for all its cops with something like `AllCops/DisabledByDefault: false`, and that we should find that, turn it off, and then go back to expressly enabling them. 

context: With standard we're conscientious about disabling-by-default and expressly enabling cops so that underlying rubocop{-performance} upgrades don't break us as often.

Reply:
We're requiring `rubocop-performance` in this file. Is your thought that some of these cops _within_ `rubocop-performance` are not enabled?

Reply:
Sorry, my point is that we have set up rubocop to explicitly enable each cop. We don't leave them all turned on by default—I'm sure we'd want to continue doing that with rubocop-performance's cops as well

Reply:
Understood, will change.